src/components/stegvisning/Samtykke/Samtykke.tsx

```diff
@@ -109,15 +109,15 @@ export function Samtykke({
           type="button"
           className={styles.button}
           variant="secondary"
-          onClick={onPrevious}
+          onClick={wrapLogger('button klikk', { tekst: 'Tilbake' })(onPrevious)}
```

Comment:
Jeg lurer på om det kan være enklere å vedlikeholde om gjenbruker translations og sender dem som "tekst" argument.
Scenario som jeg er for meg er at design oppdaterer teksten (F.eks fra "tilbake" til "avbryt") men så henger det i amplitude med "Tilbake".

Reply:
Da vil vel teksten være forskjellig basert på spårket som er valgt av bruker.  Da vil det bli vanskelig å sammenstille målinger.

Reply:
Ser den.